includes/Core/Modules/Modules.php

```diff
@@ -963,6 +969,10 @@ private function get_active_modules_option() {
 		$option = $this->options->get( self::OPTION_ACTIVE_MODULES );
 
 		if ( is_array( $option ) ) {
+			if ( in_array( 'analytics', $option, true ) ) {
+				$option[] = 'analytics-4';
+			}
+
```

Comment:
We should update this to work regardless of which option the value comes from (see legacy fallback below).

Reply:
Updated.